.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Run Tests
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+  pull_request:
+
+jobs:
+  test:
+    name: Test (${{ matrix.os }} - ${{ matrix.arch }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ macOS-latest, ubuntu-latest, windows-latest ]
+        arch: [ x64 ]
+        include:
+          - os: macOS-latest
+            arch: arm64
+    steps:
+      - name: Fix git checkout line endings
```

Comment:
Do we want to run lint only once on Ubuntu rather than hacking the line endings on Windows?

Reply:
Yeah, that's probably a better approach, I'll amend this 🙇‍♀️